sdk/helper/metrics/metrics.go

```diff
@@ -0,0 +1,46 @@
+package metrics
+
+import (
+	"sync/atomic"
+	"time"
+
+	m "github.com/armon/go-metrics"
+)
+
+// defaultLabels are the label set that should be applied to every data point
+// emitted. Use an atomic value so that we protect against concurrent access in
+// situations where the default labels are being updated after telemetry
+// initialization.
+var defaultLabels atomic.Value
+
+// Label is a wrapper around m.Label so the autoscaler doesn't have to juggle
+// importing both packages when emitting metrics.
+type Label = m.Label
+
+// SetDefaultLabels sets defaultLabels with the configured default set of
+// labels.
+func SetDefaultLabels(labels []Label) { defaultLabels.Store(labels) }
+
+// SetGauge wraps m.SetGaugeWithLabels and sets the default labels on the
+// emitted metric.
+func SetGauge(key []string, val float32) {
```

Comment:
Would it be worth adding a `SetGaugeWithLabels`?

Reply:
I created the functions based on the autoscaler's current usage of the go-metrics package and was of the opinion we could just add new ones as/when we need them.